tutor/templates/build/openedx/Dockerfile

```diff
@@ -6,6 +6,7 @@ ENV DEBIAN_FRONTEND=noninteractive
 RUN apt update && \
     apt install -y build-essential curl git language-pack-en
 ENV LC_ALL en_US.UTF-8
+{{ patch("openedx-dockerfile-minimal") }}
```

Comment:
:+1: can you please add a line to the CHANGELOG.md file describing the addition of the patch?

Reply:
sure. consider it done